.github/workflows/conformance-multicluster.yaml

```
@@ -278,11 +278,7 @@ jobs:
         if: ${{ failure() }}
         run: |
           cilium status --context ${{ steps.contexts.outputs.context1 }}
-          # FIXME: Use of timeout is a workaround for cilium/cilium-cli#384
-          timeout 5s cilium clustermesh status --context ${{ steps.contexts.outputs.context1 }}
           cilium status --context ${{ steps.contexts.outputs.context2 }}
-          # FIXME: Use of timeout is a workaround for cilium/cilium-cli#384
-          timeout 5s cilium clustermesh status --context ${{ steps.contexts.outputs.context2 }}
```

We still want to dump the status, but thanks to cilium/cilium-cli#384 it won't block forever anymore, so we can just remove the timeout:

```suggestion
cilium clustermesh status --context ${{ steps.contexts.outputs.context2 }}
```

---

:facepalm: thanks, changed

---

Note, this doesn't seem to be fixed on the branch yet.

---

:facepalm: pushed the wrong copy of the branch. Should now be fixed.